pkg/diskboot/device.go

```diff
@@ -59,6 +94,32 @@ func FindDevice(devPath string) (*Device, error) {
 	return mountDevice(devPath, fstypes)
 }
 
+func mountDeviceRW(devPath string, fstypes []string) (*Device, error) {
+	mountPath, err := ioutil.TempDir("/tmp", "boot-")
+	if err != nil {
+		return nil, fmt.Errorf("failed to create tmp mount directory: %v", err)
+	}
+
+	// unix.MS_RDONLY = 1
```

Comment:
I'm not sure what this comment is saying but it seems important, can you fill it in a bit.

Reply:
this comment basically compares the newly added fn with the existing one.
New fn: mountDeviceRW
Old Fn: mountDevice.

The only difference between these two functions is the last parameter
mount.Mount(devPath, mountPath, fstype, "", 0)<-----------
passed to the mount command. 

the existing function (mountDevice) passes unix.MS_RDONLY which is 1
wheras the new fn we add(mountDeviceRW) passes 0.